metrics/sources/summary/summary.go

```diff
@@ -273,8 +273,20 @@ func (this *summaryMetricsSource) decodeEphemeralStorageStatsForContainer(metric
 		glog.V(9).Infof("missing storage usage metric!")
 		return
 	}
-	usage := *rootfs.UsedBytes + *logs.UsedBytes
-	this.addIntMetric(metrics, &MetricEphemeralStorageUsage, &usage)
+	usage := addUsage(rootfs.UsedBytes, logs.UsedBytes)
+	if usage != nil {
+		this.addIntMetric(metrics, &MetricEphemeralStorageUsage, usage)
+	}
+}
+
+func addUsage(first, second *uint64) *uint64 {
```

Comment:
Would `safeSum` be a better name? This has nothing to do with "usage", per se...

Reply:
addUsage it's what is used in the kubelet file I copied this from. In any case, you're right it doesn't only work for usage, but I don't know that it generally applies for every sum.